lib/xenopsd.ml

```diff
@@ -38,6 +38,7 @@ let default_vbd_backend_kind = ref "vbd"
 let ca_140252_workaround = ref false
 
 let additional_ballooning_timeout = ref 120.
+let vif_ready_timeout = ref 10
```

Comment:
Make sense, because the default IGMP querier interval is 120s, so we need not wait longer than that.